### ISSUE TITLE
Keyboard focus isn't set to the first MenuItem if the Menu has a title

### ISSUE THREAD

**Issue by acmertz** [videojs/video.js#6912]
## Description
If a Menu component has a title, keyboard focus doesn't get set to the first MenuItem when the menu is triggered via keyboard.

Reduced test case: https://codepen.io/acmertz/pen/oNLZyzK

## Steps to reproduce
1. Start playing a video that has chapters (the default MenuButton for chapters includes a title in its Menu).
2. Tab over to the chapters MenuButton and trigger it via Space, Enter, or the Up/Down arrow keys.
3. Try to navigate through the MenuItems via the Up/Down arrow keys, and see that focus doesn't move between menu items.

### Expected
Up/Down arrow keys move focus between different menu items.

### Actual
Nothing happens - focus cannot be moved onto any of the MenuItems via either the Up/Down arrow keys or the Tab key.

### Error output
None

## Additional Information
When a Menu _doesn't_ have a title, everything works as expected and focus is set to the first MenuItem when the Menu is invoked via the keyboard. The issue only occurs when the Menu has a title.

The code checks for the presence of an HTML element representing the title by getting the first MenuItem and comparing its `className` to the one used for a title element. However, the check always resolves to `false` because all the MenuItem children are Video.js components and therefore don't have a `className` property.

https://github.com/videojs/video.js/blob/1cbea93b3ef16d5a900e165f3fa153771d0c3cdf/src/js/menu/menu.js#L262-L263

This appears to be left over from #3612 (#5722), when title switched from using a plain HTML element to using a Video.js component.

#### videojs
7.10.1

#### browsers
Chrome, Firefox

#### OSes
Windows

### plugins
None

**Comment by gkatsev** [comment]
Thanks, that makes sense. We'll get to reviewing the PR next week probably.
